### ISSUE TITLE
Xamarin community live stream notification.

### ISSUE THREAD

**Issue by DanielCauser** [planetxamarin/planetxamarin#568]
**Is your feature request related to a problem? Please describe.**
I hate how there are so many live streams and events that I want to be part of which I miss, often I will only see the announcements/notifications a few hours after the streams are over.

**Describe the solution you'd like**
I wonder if we could add a specific type of notification to @PlanetXamarin, just for live stream events in twitch and youtube. 
Maybe a type of notifications that get fired a few hours before the stream and another one as soon as the stream starts. For both platforms Youtube and Twitch.

**Comment by ahoefling** [comment]
I love this idea, but there needs to be a way to filter out non-xamarin streams for people that cross multiple domains. For example I stream Xamarin and non-xamarin, my stream shouldn't be notified and promoted by planet xamarin if I am doing website work.

**Comment by DanielCauser** [comment]
We have to investigate how YouTube and twitch feeds works, I know that in YouTube you can tag your videos.

**Comment by ahoefling** [comment]
I just took a look at my Twitch Video Producer dashboard. It looks like I can't create custom tags, but there is a `Mobile Development` tag. However, we could tell the streamers they would have to include `Xamarin` in the title which we should be able to filter out.

**Comment by jamesmontemagno** [comment]
We are now calling it "Xamarin Community Standup" but anything with "Xamarin" in the title would be good.

**Comment by DanielCauser** [comment]
Hey @jamesmontemagno, the idea would be not only to add the "Xamarin Community Standup" from .Net Foundation, but also other streams from any Planet Xamarin Community Author, such as the Toronto Mobile Developers user group, Melbourne Mobile Developers user group, @ahoefling, and yourself.

**Comment by jamesmontemagno** [comment]
We need to think about adding different sections on the front page for this. 

# Podcasts
* The Xamarin Podcast (Already added, needs new art)
* Merge Conflict (Already Added)
* Gone Mobile (Already Added)

# Shows
* The Xamarin Show (main feed for everything)
* Xamarin.Forms 101 - https://channel9.msdn.com/Tags/xamarin.forms-101
* Xamarin.Essentials API of the Week - https://channel9.msdn.com/Tags/xamarin.essentials
* Xamarin Community Standup - https://s.ch9.ms/Shows/NET-Community-Standups/feed

# Live Streams
* Toronto Mobile Developers Group 
* Melbourne Mobile Developer User Group

(we can get feed via: https://danielmiessler.com/blog/rss-feed-youtube-channel/) We should only bring in very select YouTube channels though as we don't want everyone to be submitting their personal feeds.

I want to remove the "Open-source Software" as we have builtwithdot.net for that and it doesn't belong there unless we have a larger strategy for that.

Thoughts here?

**Comment by ahoefling** [comment]
@jamesmontemagno this is a great documentation and I love this!

I think the original question @DanielCauser had was making visibility to personal xamarin streamers. For example Daniel expressed interest in attending my Twitch Stream after the fact and he missed out on my Visual Challenge live. Would the intent of Live Streams to not add personal Twitch or YouTube channels for Xamarin live coders?

**Comment by jamesmontemagno** [comment]
Yeah, maybe we can do "Streamers" basically saying these are live code streamers that regularly stream Xamarin content.

**Comment by dansiegel** [comment]
Just a thought here but what about introducing some new interfaces and expanding PlanetXamarin a bit.

```cs
public interface IAmATwitchStreamer
{
    string TwitchStreamUrl { get; }
}

public interface IAmAYouTubeChannel
{
    string YouTubeChannelUrl { get; }
}
```

I'm probably bias but I do think it would be good for the community to be able to more easily find people who are streaming or posting series on YouTube

**Comment by Cheesebaron** [comment]
@dansiegel What if author streams on Mixer or some other platform? Maybe we could make this more generic?

Also how do we want to present this on the webpage? Show twitch, youtube or platform x icon next to the author?

**Comment by dansiegel** [comment]
That's a great question. I could see this really varying quite a bit based on how we all think it should be presented. I suppose it could be more generic where it's more of an `IAmAStreamer` with an enumeration of streams. Perhaps the answer is that when someone is a streamer we add the icon for the platform with the link to their channel on their card like what I have below. I do feel this is still niche enough that we may want to do something like we do with Podcasts and also separate out streamers somewhere other than the authors so that people can easily find Xamarin streamers and get to their channels to follow.

![image](https://user-images.githubusercontent.com/3860573/62479769-0bda1e00-b763-11e9-976c-262513b847dc.png)
